### PR TITLE
Updates tgui and build backend, fixes the build.bat for those with latest eslint (?)

### PR DESCRIPTION
**Pull request by silicons**
## About The Pull Request

All I know is this fixed it and doesn't seem to be causing issues so.

And yes, this ports in a bunch of interfaces we aren't using, but imo it doesn't hurt.

### CODE REVIEW

**Comment by SandPoot** [comment]
Please check out the stuff that still need to use citadel's version.

File: tgui/packages/tgui/interfaces/CrewManifest.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+import { useBackend } from "../backend";
+import { Icon, Section, Table, Tooltip } from "../components";
+import { Window } from "../layouts";
+
+const commandJobs = [
+  "Head of Personnel",
+  "Head of Security",
+  "Chief Engineer",
+  "Research Director",
+  "Chief Medical Officer",
```
Comment (SandPoot) [inline tgui/packages/tgui/interfaces/CrewManifest.js:11]: Someone's missing

Comment (SandPoot) [reply tgui/packages/tgui/interfaces/CrewManifest.js:11]: Actually, don't mind, all looks good for me, if someone's going to touch it later, they can change it.

**Comment by silicons** [comment]
#14681

**Pull request merged**